app/services/aws_sqs_poller.rb

```diff
@@ -7,30 +7,38 @@ def self.poll
 
     poller.poll do |msg|
       message_body = JSON.parse(msg[:body])
-      next unless message_body["job_instance"]["name"] == Rails.configuration.alma[:full_dump_job_name]
-      dump = AlmaFullDumpFactory.full_bib_dump(message_body)
+      job_name = message_body["job_instance"]["name"]
+      next unless jobs_map.keys.include?(job_name)
+      dump = AlmaDumpFactory.bib_dump(
+        message_body,
+        jobs_map[job_name]
+      )
       # running dump creation in the background prevents the queue
       # event from timing out and requeuing
-      AlmaFullDumpTransferJob.perform_later(
+      AlmaDumpTransferJob.perform_later(
         dump: dump,
         job_id: message_body["job_instance"]["id"]
       )
     end
   end
 end
 
-class AlmaFullDumpFactory
+def jobs_map
```

Comment:
Is this out of the class definition on purpose?

Reply:
oops thank you for catching this